docs/index.md

```diff
@@ -1,21 +1,21 @@
 # ReBloom - Bloom Filter Module for Redis
 
-This provides a scalable bloom filter as a Redis data type. Bloom filters
-are probabilistic data structures that do a very good job at quickly
+This module provides a scalable bloom filter as a Redis data type. Bloom filters
+are probabilistic data structures that do a very good job of quickly
 determining if something is contained within a set.
 
 ## Building
 
-In order to use this module, build it using `make` and load it into Redis.
+Use the `make` command to build and load this module into Redis.
```

Comment:
This is actually wrong. It implies that `make` will load the module into Redis, which is incorrect. The module must be manually loaded into Redis.